tests/test_multidb.py

```diff
@@ -0,0 +1,36 @@
+from unittest import expectedFailure
+
+from django.test import TestCase
+
+from admin_interface.models import Theme
+
+from .routers import DatabaseAppsRouter
+
+
+class AdminInterfaceModelsWithDBRoutingTestCase(TestCase):
+    databases = ["replica"]
+
+    def test_standard_dbrouter(self):
+        router = DatabaseAppsRouter()
+        db_for_theme = router.db_for_read(Theme)
+        assert db_for_theme == "default"
+
+    def test_dbrouter_selects_correct_db(self):
+        DATABASE_APPS_MAPPING = {
+            "admin_interface": "replica",
+        }
+        router = DatabaseAppsRouter(db_map=DATABASE_APPS_MAPPING)
+        db_for_theme = router.db_for_read(Theme)
+        assert db_for_theme == "replica"
+
+    @expectedFailure
+    def test_dbrouter_errors_when_fetching_from_default(self):
+        Theme.get_active_theme()
```

Comment:
This is not an expected failure (a test that doesn’t work at the moment but maybe updating something in the future will make it work), but a test for code that should fail, so it should use `assertRaises`

Reply:
I've always interpreted `expectedFailure` literally. I've updated it.